pkg/azureclients/armclient/azure_armclient.go

```diff
@@ -95,17 +122,23 @@ func New(authorizer autorest.Authorizer, clientConfig azureclients.ClientConfig,
 		apiVersion:       apiVersion,
 		regionalEndpoint: fmt.Sprintf("%s.%s", clientConfig.Location, url.Host),
 	}
+	client.client.Sender = autorest.DecorateSender(client.client.Sender, sendDecoraters...)
```

Comment:
`client.client.Sender` is `nil` here. Should this be `client.client`?

Reply:
It is initialized on line 83, so it shouldn't be nil.

Reply:
Right. Missed that.